h2o-py-cloud-extensions/setup.py

```diff
@@ -60,7 +60,7 @@
         "Topic :: Scientific/Engineering :: Information Analysis",
 
         # Pick your license as you wish (should match "license" above)
-        "License :: OSI Approved :: Apache Software License",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
```

Comment:
Hey, you have been referring to GNU Affero General Public License v3.0 in the files above

and here is GNU Affero General Public License v3.  Please stick with one, either v3.0 or v3.

Reply:
This is the list of licenses accepted by PyPi.:
https://autopilot-docs.readthedocs.io/en/latest/license_list.html

So I will change to v3 everywhere

Reply:
Sounds good.